noisy-simulator/src/density_matrix_simulator/tests.rs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+use super::DensityMatrixSimulator;
+use crate::tests;
+
+#[test]
+fn measure_0() {
```

Comment:
One of the points we talked about offline was naming tests. In general, it's helpful for reviewers and maintainers if tests convey what they are checking in the name, even if it is verbose.

For example, this test could be named something like `measuring_plus_state_yields_zero_with_50_percent_probability`.

Reply:
I changed all of the unit tests names to be more descriptive.